gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadOptions.java

```diff
@@ -39,7 +39,7 @@ public enum Fadvise {
   public static final double DEFAULT_BACKOFF_MULTIPLIER = 1.5;
   public static final int DEFAULT_BACKOFF_MAX_INTERVAL_MILLIS = 10 * 1000;
   public static final int DEFAULT_BACKOFF_MAX_ELAPSED_TIME_MILLIS = 2 * 60 * 1000;
-  public static final boolean DEFAULT_FAST_FAIL_ON_NOT_FOUND_ENABLED = true;
+  public static final boolean DEFAULT_FAST_FAIL_ON_NOT_FOUND_ENABLED = false;
```

Comment:
is this a breaking change? Should this be part of release notes?

Reply:
We will do it as part of new release. Is there any section for release notes.

Reply:
Usually we include docs and release notes changes in the same PR as change itself:
- https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcs/CHANGES.md
- https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcs/CONFIGURATION.md#io-configuration